charts/lagoon-core/values.yaml

```
@@ -639,7 +639,7 @@ sshPortal:
   enabled: false
   replicaCount: 2
   image:
-    repository: amazeeiolagoon/ssh-portal
+    repository: uselagoon/ssh-portal
```

pr-branches are only in testlagoon
```suggestion
    repository: testlagoon/ssh-portal
```